baselines/fedmix/fedmix/conf/cifar10.yaml

```diff
@@ -0,0 +1,33 @@
+---
+
+seed: 42
+device: cuda
+num_gpus: 1
```

Comment:
For consistency with other baselines, could you specify it like:
```
client_resources:
    num_cpus: <something>
    num_gpus: <something>
```

Reply:
As a side note, `num_gpus` specifies the ratio of GPU VRAM a client should have access to. So for instance if you set `num_gpus=0.2` it means that you are ok with `1/0.2 = 5` clients to concurrently run on the GPU. I'm bringing this up because your VGG model is not that big  (~1GB of VRAM) and you can easily pack more in a single GPU. More details about this in the docs: https://flower.dev/docs/framework/how-to-run-simulations.html#virtualclientengine-resources